### PR TITLE
fix: Capacitor bindings wallet plugin

### PR DESCRIPTION
**Pull request by amadeu2**
## Summary
Android project as incorrect folder names as Java conventions, producing errors at build time.
On Desktop the name of Stronghold directory name is `__storage__`, we currently are using `database` on the plugin code, so will change it.
Also added error handling to avoid crashes on Java code and update Capacitor plugin dependencies to latest version.

### Changelog
```
 change to use storage folder as in desktop
 handle exceptions on android
 fix rename directory to new package name on android
 update capacitor core deps to latest version
```

## Relevant Issues
Closes https://github.com/iotaledger/firefly/issues/2464

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [x] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [x] iOS
	- [x] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/backend/bindings/capacitor/package.json [hunk 1]
```diff
@@ -38,16 +38,16 @@
     "@capacitor/core": "^3.4.0"
   },
   "devDependencies": {
-    "@capacitor/android": "^3.4.0",
-    "@capacitor/core": "^3.4.0",
-    "@capacitor/ios": "^3.4.0",
+    "@capacitor/android": "^3.4.1",
+    "@capacitor/core": "^3.4.1",
+    "@capacitor/ios": "^3.4.1",
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/package.json:43]: The `yarn.lock` might be out of date after these changes

File: packages/backend/bindings/capacitor/android/src/main/java/org/iota/fireflyactorsystem/WalletPlugin.java [hunk 2]
```diff
@@ -60,7 +68,7 @@ public void sendMessage(final PluginCall call) {
     @PluginMethod()
     public void destroy(final PluginCall call) {
         if (!isInitialized) {
-            call.reject("Wallet is not initialized yet");
+            call.resolve()
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/android/src/main/java/org/iota/fireflyactorsystem/WalletPlugin.java:71]:
```suggestion
            call.resolve();
```

File: packages/backend/bindings/capacitor/package.json (hunk 1 above)
Comment (amadeu2) [reply packages/backend/bindings/capacitor/package.json:43]: thanks will update it

**Comment by amadeu2** [comment]
yarn lint does not fix any file

**Comment by rajivshah3** [comment]
> yarn lint does not fix any file

This error is coming from prettier, so `yarn format` should fix it

**Comment by amadeu2** [comment]
I've been executed yarn format too but same 🤔

**Comment by rajivshah3** [comment]
Oooh this one is coming up for me on `develop`:

```
[error] packages/shared/components/Text.svelte: Valid <svelte:...> tag names are svelte:head, svelte:options, svelte:window, svelte:body, svelte:self or svelte:component
[error]   4 |
[error]   5 | <span class="text-component">
[error] > 6 |     <svelte:element this={type} class={`${customClassesString} ${classes}`}>
[error]     |     ^
[error]   7 |         <slot />
[error]   8 |     </svelte:element>
[error]   9 | </span> (6:5)
[error] 4: 
[error] 5: <span class="text-component">
[error] 6:     <svelte:element this={type} class={`${customClassesString} ${classes}`}>
[error]         ^
[error] 7:         <slot />
[error] 8:     </svelte:element>
```

**Comment by rajivshah3** [comment]
Ah nvm it might be from old dependencies

**Comment by amadeu2** [comment]
cloned in a new folder, installed from clean, fixed format and pushed, in my console I get this:
```bash
 fix/capacitor-wallet-plugin $ yarn format-check
yarn run v1.22.17
$ prettier -c "**/*.{ts,js,json,scss,css,svelte}"
Checking formatting...
All matched files use Prettier code style!
✨  Done in 6.47s.
```
but seems here's some I missing 🤔

**Comment by rajivshah3** [comment]
The CI is running on the code after it's merged with `develop`. It seems like there's issues on `develop`. I can fix them in a separate PR

**Comment by amadeu2** [comment]
ahh yes I blame myself 😬, this was from the previous merged PR, thanks! 😊

File: packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift [hunk 3]
```diff
@@ -15,9 +15,9 @@ public class WalletPlugin: CAPPlugin {
             }
             let fm = FileManager.default
             let documents = fm.urls(for: .documentDirectory, in: .userDomainMask).first!
-            let path = documents.appendingPathComponent("database", isDirectory: true).path
+            let path = documents.appendingPathComponent("__storage__", isDirectory: true).path
             if !fm.fileExists(atPath: path) {
-                try fm.createDirectory(atPath: path, withIntermediateDirectories: false, attributes: nil)
+                try fm.createDirectory(atPath: path, withIntermediateDirectories: true, attributes: nil)
             }
             call.keepAlive = true
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift:22]: Should we release this like we do on Android? I think there's a `TODO` about this on line 49

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**